microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls.md

```diff
@@ -123,24 +127,31 @@ To test the feature, send yourself a URL in a chat message app, or a desktop ema
 <!-- ====================================================================== -->
 ## Demo
 
-[DevTools Tips](https://devtoolstips.org/) is a PWA that handles URLs to its own domain so that the app opens instead of the website when one is used.
+[DevTools Tips](https://devtoolstips.org/) is a demo Progressive Web App (PWA) that handles URLs for its own domain, so that the app opens instead of the website when a URL for its domain is used.
 
-To test URL handling on DevTools Tips:
+To test URL handling on the DevTools Tips demo:
 
-*  [Enable the feature](#enable-url-handling) in Microsoft Edge.
-*  Go to [DevTools Tips](https://devtoolstips.org/).
-*  Install the app locally.
-*  Press `Windows logo key` + `R` to open the Windows Run app.
-*  Enter a URL to one of the tips on the site such as https://devtoolstips.org/tips/en/find-css-changes/
-*  Press `Enter`.
+1. [Enable URL handling](#enable-url-handling) in Microsoft Edge.
+
+1. Go to the [DevTools Tips](https://devtoolstips.org/) PWA.
+
+1. Click the **App available. Install DevTools Tips** (![The 'App available. Install DevTools Tips' icon.](../media/app-available-icon.png)) icon in the Address bar to install the DevTools Tips app locally.
+
+1. Press `Windows logo key` + `R` to open the Windows **Run** dialog.
+
+1. Enter a URL for one of the tips on the site, such as https://devtoolstips.org/tips/en/find-css-changes/.
+
+1. Press `Enter`.
 
-Windows knows that your app is registered to handle this URL and asks you to choose which app you want to use. Select the DevTools Tips apps. You can also select **Remember my choice** to avoid seeing this dialog every time.
+Windows knows<!--if it knows, why does it ask?--> that your app is registered to handle this URL, and asks you to choose which app you want to use. Select the **DevTools Tips** app. If you want to avoid seeing this dialog every time, select **Remember my choice**.
```

Comment:
If Windows knows, why does it ask?  Clarify logic of sentence.

Reply:
It asks because there may be more than 1 app registered to handle URLs in the OS. I'll clarify the sentence, thanks.